test/library/library.go

```diff
@@ -18,6 +18,10 @@ var (
 	WaitPollTimeout  = 10 * time.Minute
 )
 
+type TestingT interface {
+	Logf(format string, args ...interface{})
```

Comment:
nit: this interface doesn't implement all of testing.T, just logging, maybe naming could be differently.

Reply:
done.